src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -512,12 +518,20 @@ public static void configCheck(final ConfigType type) {
 			plantBlocks.add(Material.SUNFLOWER.toString());
 			plantBlocks.add(Material.POPPY.toString());
 			plantBlocks.add(Material.FERN.toString());
+			plantBlocks.add(Material.LILY_OF_THE_VALLEY.toString());
+			plantBlocks.add(Material.WITHER_ROSE.toString());
+			plantBlocks.add(Material.CORNFLOWER.toString());
 			plantBlocks.add(Material.LARGE_FERN.toString());
 			plantBlocks.add(Material.RED_MUSHROOM.toString());
 			plantBlocks.add(Material.RED_MUSHROOM_BLOCK.toString());
 			plantBlocks.add(Material.BROWN_MUSHROOM.toString());
 			plantBlocks.add(Material.BROWN_MUSHROOM_BLOCK.toString());
 			plantBlocks.add(Material.MUSHROOM_STEM.toString());
+			plantBlocks.add(Material.WARPED_ROOTS.toString());
+			plantBlocks.add(Material.CRIMSON_ROOTS.toString());
+			plantBlocks.add(Material.TWISTING_VINES_PLANT.toString());
+			plantBlocks.add(Material.WEEPING_VINES_PLANT.toString());
+			plantBlocks.add(Material.NETHER_SPROUTS.toString());

```

Comment:
Still don't think these should be plantbendable by default

Reply:
Looks like a majority of server owners are going to make them default anyways, we might as well add them.